### PR TITLE
Ip unit tests implementation & coding convention update.

### PR DESCRIPTION
**Pull request by nvthongswansea**

### CODE REVIEW

File: ip.go [hunk 1]
```diff
@@ -57,83 +62,128 @@ type IpCreateResponse struct {
...
+func (c *Client) GetIpList() ([]Ip, error) {
 	r := Request{
 		uri:    apiIpBase,
-		method: "GET",
+		method: http.MethodGet,
 	}
 
-	response := new(Ips)
+	var response Ips
+	var list []Ip
```
Comment (itakouna) [inline ip.go]: the `list` is a generic name. Plz, use name reflects the content of the variable e.g., IPs.

File: ip.go [hunk 2]
```diff
@@ -57,83 +62,128 @@ type IpCreateResponse struct {
...
 
-//Returns 0 if an error was encountered
+//GetIpEventList gets a list of an IP's events
+func (c *Client) GetIpEventList(id string) ([]IpEvent, error) {
+	r := Request{
+		uri:    path.Join(apiNetworkBase, id, "events"),
+		method: http.MethodGet,
+	}
+	var response IpEventList
+	var list []IpEvent
```
Comment (itakouna) [inline ip.go]: same here `list`

File: ip_test.go [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+		UserUuid:      dummyUuid,
+	}}
+	return mock
+}
+
+func prepareIpEventListHTTPGet() string {
+	event := getMockIpEvent()
+	res, _ := json.Marshal(event.Properties)
+	return fmt.Sprintf(`{"events": [%s]}`, string(res))
+}
```
Comment (itakouna) [inline ip_test.go]: a new line is missing.

**Comment by itakouna** [comment]
would you add the status of Travis into README file? just put the following line into README
```
[![Build Status](https://travis-ci.com/gridscale/gsclient-go.svg?branch=master)](https://travis-ci.com/gridscale/gsclient-go)
```

File: ip_test.go (hunk 3 above)
Comment (nvthongswansea) [reply ip_test.go]: What do you mean by "a new line is missing", please?

**Pull request merged**